lively.morphic/rendering/property-dom-mapping.js

```diff
@@ -11,9 +11,14 @@ import bowser from 'bowser';
  * @returns {Node} the DOM node with changed style properties.
  */
 export function stylepropsToNode (styleProps, node) {
+  node.style.removeProperty('padding-left');
+  node.style.removeProperty('padding-right');
+  node.style.removeProperty('margin-bottom');
+  node.style.removeProperty('margin-top');
   for (let prop in styleProps) {
     node.style[prop] = styleProps[prop];
   }
+  // FIXME: remove the props NOT mentioned in styleProps but still mentioned in style?
```

Comment:
> ist das richtig das stylepropsToNode nur props hinzufügt oder überschreibt aber keine löscht? Also ist das ein feature oder ein bug?

This is an interesting question!

I had not thought about this previously, which I think is a byproduct of us setting "default values" (i.e., what CSS would default to) quite excessively, so that e.g., a removed fill color seemed to work because it was explicitly overridden (might not be the best example, I hope you catch my drift).

Hovewer, I believe you are right and we should delete unset properties here. I'd propose to just set `node.style = {}` at the beginning of this function? In case this breaks stuff, I think we would have a bug in places where we populate `styleProps`, as I'd expect this to be an idempotent operation.
However, we would still have the possibility to go the hacky route of explicitly deleting properties here, in case tracing problems in the correct solution gets to confusing.

**TL;DR:** I think this should delete all style properties that are set on `node` but are not specified in `styleProps`.